src/app/edit.component/edit.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core'
+import {DashboardComponent} from '../dashboard.component/todo.dashboard'
+import {HeroService} from '../services/todo.service'
+import { RouteConfig, ROUTER_DIRECTIVES, ROUTER_PROVIDERS } from '@angular/router-deprecated';
+import {EditComponent} from '../edit.component/edit.component';
+import * as _  from 'lodash';
+
+@RouteConfig([
```

Comment:
multiple same routing ?

Reply:
this comment i am not understand

Reply:
You written routing already in app.component.ts